site/reference/fn/sink/README.md

```diff
@@ -1,51 +1,45 @@
 ---
-title: "Sink"
+title: "`sink`"
 linkTitle: "sink"
 type: docs
 description: >
-   Specify a directory as an output sink package
+   Write resources to a local directory
 ---
 
 <!--mdtogo:Short
-    Specify a directory as an output sink package
+   Write resources to a local directory
 -->
 
-Implements a [sink function] by reading STDIN and writing configuration.
+`sink` reads resources from `stdin` and writes them to a local directory.
+It is useful for chaining functions using Unix pipe. For more details, refer to
+[Chaining functions].
 
-Sink will not prune / delete files for delete resources because it only knows
-about files for which it sees input resources.
-
-### Examples
+### Synopsis
 
-<!--mdtogo:Examples-->
+<!--mdtogo:Long-->
 
 ```shell
-# run a function using explicit sources and sinks
-kpt fn source DIR/ |
-  kpt fn run --image gcr.io/example.com/my-fn |
-  kpt fn sink DIR/
+kpt fn sink DIR [flags]
+
+DIR:
+  Path to a local directory to resources to. Directory must exist.
```

Comment:
> Path to a local directory to resources to.

This doesn't read well.

Is `DIR` required? If so, this behavior is different

Reply:
missed `write` :) fixed.

Yes, `DIR` is required (probably was done as part of CLI usability work).

>if so, this behavior is different

what behavior are you referring to ? and what is different ?

Reply:
`DIR` was optional before. If unspecified, it prints to stdout.